tests/resources/__init__.py

```diff
@@ -1,16 +1,14 @@
-import os
 import json
+from importlib import resources
 
-_HERE = os.path.realpath(os.path.dirname(__file__))
+with resources.path(__package__, "summary.json") as path:
+    SUMMARY = json.loads(path.read_text())
 
-with open(os.path.join(_HERE, "summary.json")) as h:
-    SUMMARY = json.load(h)
+with resources.path(__package__, "csv_links_response.txt") as path:
+    CSV_LINKS_RESPONSE_TEXT = path.read_text()
 
-with open(os.path.join(_HERE, "csv_links_response.txt")) as h:
-    CSV_LINKS_RESPONSE_TEXT = h.read()
+with resources.path(__package__, "csv_response.csv") as path:
+    CSV_RESPONSE = path.read_text()
 
-with open(os.path.join(_HERE, "csv_response.csv")) as h:
-    CSV_RESPONSE = h.read()
-
-with open(os.path.join(_HERE, "metadata.tsv")) as h:
-    METADATA_RESPONSE = h.read()
+with resources.path(__package__, "metadata.tsv") as path:
+    METADATA_RESPONSE = path.read_text()
```

Comment:
Might as well just `resources.read_text(__package__, "resource.ext")` here if trying to simplify

Reply:
I did not know that function, I must have read right pass it on the docs